vars/handlePipelineStepErrors.groovy

```diff
@@ -1,34 +1,56 @@
+import com.cloudbees.groovy.cps.NonCPS
+
+import com.sap.piper.ConfigurationHelper
+
 import groovy.text.SimpleTemplateEngine
 import groovy.transform.Field
 
 @Field STEP_NAME = getClass().getName()
 
+@Field Set PARAMETER_KEYS = [
+    'echoDetails',
+    'stepName',
+    'stepNameDoc',
+    'stepParameters'
+]
+
 void call(Map parameters = [:], body) {
-    def stepParameters = parameters.stepParameters //mandatory
-    def stepName = parameters.stepName //mandatory
-    def verbose = parameters.get('echoDetails', true)
+    // load default & individual configuration
+    Map config = ConfigurationHelper.newInstance(this)
+        .loadStepDefaults()
+        .mixin(parameters, PARAMETER_KEYS)
+        .withMandatoryProperty('stepParameters')
+        .withMandatoryProperty('stepName')
+        .addIfEmpty('stepNameDoc' , parameters.stepName)
```

Comment:
This is new isn't it?
This is not used somewhere, isn't it?

Reply:
@OliverNocon could you explain this

Reply:
configuration is now required ...

Reply:
The question was about `stepNameDoc`, not about Configuration in general.

Reply:
Maybe it gots simply forgotten: What is the story behind stepNameDoc?

Reply:
It's used [here](https://github.com/SAP/jenkins-library/blob/master/vars/pipelineStashFilesAfterBuild.groovy#L13) to reference to a different docs page than the actual step name.